src/containers/MarketsTable/index.tsx

```diff
@@ -1,182 +1,104 @@
 import * as React from 'react';
-import { injectIntl } from 'react-intl';
-import { connect, MapDispatchToProps } from 'react-redux';
-import { RouterProps } from 'react-router';
-import { withRouter } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { useHistory } from 'react-router-dom';
 import { Decimal, TickerTable } from '../../components';
-import { IntlProps } from '../../index';
 import {
     Market,
     marketsFetch,
     marketsTickersFetch,
-    RootState,
     selectMarkets,
     selectMarketTickers,
-    selectUserLoggedIn,
     setCurrentMarket,
-    Ticker,
 } from '../../modules';
-import { rangerConnectFetch, RangerConnectFetch } from '../../modules/public/ranger';
-import { RangerState } from '../../modules/public/ranger/reducer';
+import { rangerConnectFetch } from '../../modules/public/ranger';
 import { selectRanger } from '../../modules/public/ranger/selectors';
 
-interface ReduxProps {
-    markets: Market[];
-    marketTickers: {
-        [key: string]: Ticker,
-    };
-    rangerState: RangerState;
-    loggedIn?: boolean;
-}
-
-interface DispatchProps {
-    fetchMarkets: typeof marketsFetch;
-    rangerConnect: typeof rangerConnectFetch;
-    setCurrentMarket: typeof setCurrentMarket;
-    tickers: typeof marketsTickersFetch;
-}
-
-interface State {
-    currentBidUnit: string;
-}
-
-export type Props = ReduxProps & DispatchProps & RouterProps & IntlProps;
-
-class MarketsTableContainer extends React.Component<Props, State> {
-    constructor(props: Props) {
-        super(props);
-
-        this.state = {
-            currentBidUnit: '',
-        };
+const defaultTicker = {
```

Comment:
Maybe let's move such things to constants?
Hence a component size will be smaller

Reply:
will be refactored later